File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/controllers/gains/FullStateGains.java
package org.firstinspires.ftc.teamcode.control.controllers.gains;

public class FullStateGains {

    public final double pGain, vGain, aGain;

    public FullStateGains(
            double pGain,
            double vGain,
            double aGain
    ) {
        this.pGain = pGain;
        this.vGain = vGain;
        this.aGain = aGain;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/gainmatrices/FullStateGains.kt
package org.firstinspires.ftc.teamcode.control.gainmatrices

data class FullStateGains @JvmOverloads constructor(
    @JvmField var pGain: Double = 0.0,
    @JvmField var vGain: Double = 0.0,
    @JvmField var aGain: Double = 0.0,
)
